test_pull_requests

```diff
@@ -124,12 +124,37 @@ NEEDS_REBASE_LABEL          = "needs-rebase"
 GITHUB_API_BASE_URL = 'https://api.github.com'
 GITHUB_BASE_URL = 'https://github.com'
 
+# TODO: create a :state to "content" mapping for composing new bot
+# messages
+PR_STATES = {
+  /^Evaluating/ => :evaluating,
```

Comment:
I don't like this hodgepodge of things in a map.  They are not all states for the same thing.  I also don't love the abstraction being so fuzzy.  Meaning checking for the beginning of the string but not the end.  If we are going to abstract it, you need to be more precise.  Otherwise the details of the fuzziness will be forgotten from the callers.